cclib/parser/gaussianparser.py

```diff
@@ -213,6 +214,232 @@ def extract(self, inputfile, line):
                 "package_version"
             ] = f"{self.YEAR_SUFFIXES_TO_YEARS[year_suffix]}+{revision}"
             self.metadata["platform"] = platform
+            self.metadata['date'] = datetime.datetime.strptime(run_date, "%d-%b-%Y")
+
+        # Extract number of processors, memory, keywords line, solvation,
+        # empirical dispersion and type of system (ground or transition state)
+        # ******************************************
+        # %mem=24GB
+        # %nprocshared=12
+        # Will use up to   12 processors via shared memory.
+        # %chk=Int23.chk
+        # ----------------------------------------------------------------------
+        # # uM062X/Def2TZVP guess=read geom=check opt=(calcfc,ts,noeigen,maxstep
+        # =5) freq=noraman
+        # ----------------------------------------------------------------------
+        elif '%mem' in line:
+            mem = line.strip().split('=')[-1]
+
+        elif '%nprocs' in line:
+            nprocs = int(line.strip().split('=')[-1])
```

Comment:
```suggestion
```

Memory and CPU is now handled by https://github.com/cclib/cclib/pull/1271 so we can remove from here :)